src/frontend/src/components/EditableTextInputList.tsx

```diff
@@ -94,37 +95,39 @@ const EditableTextInputList: React.FC<EditableTextInputListProps> = ({
     !readOnly ? (
       <li key={index} className={'mb-2'}>
         <Box marginBottom={1}>
-          <TextField
-            required
-            fullWidth
-            type="text"
-            id={`bullet-${index}`}
-            name={`bullet-${index}`}
-            value={item.toString()}
-            label={label}
-            placeholder="Input new bullet here..."
-            ref={isLastElement(index) ? focusRef : null}
-            autoFocus={hasTyped && isLastElement(index)}
-            onKeyDown={(e: any) => handleKeyDown(e, index)}
-            onChange={(e: any) => {
-              setHasTyped(true);
-              update(index, e.target.value);
-              if (isLastElement(index)) {
-                setLastInput(e.target.value);
-              }
-            }}
-            InputProps={{
-              endAdornment: (
-                <IconButton
-                  aria-label="delete"
-                  disabled={disabledItems && disabledItems[index]}
-                  onClick={() => removeButtonOnClick(index)}
-                >
-                  <DeleteIcon />
-                </IconButton>
-              )
-            }}
-          />
+          <Typography>
+            <TextField
```

Comment:
does TextField really need to be wrapped in Typography? I thought Typography was just for bare text...